testing/tests/DevExpress.ui.widgets.editors/tagBox.tests.js

```diff
@@ -3960,6 +4029,75 @@ QUnit.module('searchEnabled', moduleSetup, () => {
 
         assert.strictEqual(loadStub.callCount, 3);
     });
+
+    QUnit.test('TagBox should add all clicked items after search if dataSource is async (T958611)', function(assert) {
+        const $tagBox = $('#tagBox').dxTagBox({
+            dataSource: getDSWithAsyncSearch(),
+            valueExpr: 'id',
+            displayExpr: 'id',
+            showSelectionControls: true,
+            searchEnabled: true,
+            searchExpr: 'id',
+            searchTimeout: TIME_TO_WAIT,
+            opened: true
+        });
+        const tagBox = $('#tagBox').dxTagBox('instance');
+
+        this.clock.tick(TIME_TO_WAIT * 3);
```

Comment:
Why do we use **3** times to wait here?

Reply:
I use 2 * TIME_TO_WAIT in dataSource load timeout. Here I need the load function is definitely completed.